grails-app/i18n/messages_et.properties

```diff
@@ -347,6 +374,7 @@ assertions.warnings = hoiatusega
 assertions.passed = läbinud
 assertions.missing = puudu
 assertions.unchecked = testimata
+show.btn.cancel = Cancel
```

Comment:
Cancel is also in commons

Reply:
Changing this at the moment would mean changing bie-plugin also. And there are a lot more duplicated things in here which I don't have time for at the moment